### DISCUSSION TITLE
Nuxt 3 public runtime configs are now in $config.public instead of root

### DISCUSSION THREAD

**Discussion by hecktarzuli** [nuxt/nuxt#16818] [General]
When I migrated my site to nuxt 3 via Nuxt Bridge, runtime configs are now off of a property called `public` instead of root.
```
// nuxt.config.js
publicRuntimeConfig: {
  foo: 'bar'
}
```
is now under `$config.public.foo` instead of `$config.foo`

Is this a bug?  If not, it should be included in the [migration guide](https://v3.nuxtjs.org/bridge/overview/#feature-flags).
cc @danielroe

**Comment by danielroe** [comment]
It's not a bug, and yes, should be included in the migration guide. 😊

**Reply by hecktarzuli** [reply]
Docs Update: https://github.com/nuxt/framework/pull/7775
